index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Esmatullah</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="navbar">
+            <div class="logo">
+                <h3>My Logo</h3>
+            </div>
+            <div class="menue">
+                <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
+            </div>
+        </div>
+    </header>
+    <div class="info">
+        <div class="container">
+            <h1>I’m Esmatullah Glad to see you!</h1>
+        </div>
+        <div>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a class="connect-btn" href="">LET’S CONNECT</a>
+            <img src="icon/social_media_icon.png" alt="">
```

Comment:
- [ ] Please, ensure that your `img` tag always has an `alt` attribute this is good for accessibility reasons, for example, for a visually impaired individual or a slow internet user. 🙂